internal/workerutil/errors.go

```diff
@@ -0,0 +1,12 @@
+package workerutil
+
+import "errors"
+
+// ErrDequeueTransaction occurs when Dequeue is called from inside a transaction.
+var ErrDequeueTransaction = errors.New("unexpected transaction")
+
+// ErrDequeueRace occurs when an upload selected for dequeue has been locked by another worker.
```

Comment:
Not sure, "row"/"task"/"item"/"record"?
```suggestion
// ErrDequeueRace occurs when a row selected for dequeue has been locked by another worker.
```

Reply:
😛